src/couch_mrview_index.erl

```diff
@@ -134,15 +134,15 @@ open(Db, State) ->
                 {ok, {OldSig, Header}} ->
                     % Matching view signatures.
                     NewSt = couch_mrview_util:init_state(Db, Fd, State, Header),
-                    {ok, NewSt#mrst{fd_monitor=erlang:monitor(process, Fd)}};
+                    {ok, NewSt};
                 % end of upgrade code for <= 1.2.x
                 {ok, {Sig, Header}} ->
                     % Matching view signatures.
                     NewSt = couch_mrview_util:init_state(Db, Fd, State, Header),
-                    {ok, NewSt#mrst{fd_monitor=erlang:monitor(process, Fd)}};
+                    {ok, NewSt};
                 _ ->
                     NewSt = couch_mrview_util:reset_index(Db, Fd, State),
-                    {ok, NewSt#mrst{fd_monitor=erlang:monitor(process, Fd)}}
+                    {ok, NewSt}
```

Comment:
@tonysun83 We rather rarely catch from `case` (probably because it look ugly, me been opinionated and all that). Theoretically I can do something like `{ok, couch_mrview_util:init_state(Db, Fd, State, Header)}` in clauses, but honestly I was trying to avoid mixing refactoring with such a small change.